sdk/eventhub/azure-eventhubs/azure/eventhub/client_abstract.py

```diff
@@ -82,54 +82,54 @@ def _build_uri(address, entity):
 
 class EventHubClientAbstract(object):
```

Comment:
Shouldn't this be EventHub**s**ClientAbstract?

Reply:
All 4 languages have class EventHubClient, so for this class I think we should use EventHub instead of EventHubs